sdks/java/io/bigquery-io-perf-tests/build.gradle

```diff
@@ -17,15 +17,15 @@
  */
 
 plugins { id 'org.apache.beam.module' }
-applyJavaNature(exportJavadoc: false, publish: false)
+applyJavaNature(enableStrictDependencies:true,exportJavadoc: false, publish: false)
 provideIntegrationTestingDependencies()
 enableJavaPerformanceTesting()
 
 description = "Apache Beam :: SDKs :: Java :: Google BigQuery IO Performance tests"
 ext.summary = "Performance tests for Google BigQuery IO sources and sinks"
 
 dependencies {
-    compile library.java.google_api_services_bigquery
+    testImplementation library.java.google_api_services_bigquery
```

Comment:
As far as I know, there's not a significant difference between using testImplementation and testCompile, so let's make this testCompile to match all the other dependencies.